doc: use ncu-team sync to sync member list

MylesBorins opened the pull request:
As was discussed in the last meeting all members that did not fill out
the availability doodle have been moved to observer status.

Observers are not counted toward quorum and do not get to vote

Refs: https://github.com/nodejs/modules/issues/8#issuecomment-364261695
Refs: https://github.com/nodejs/modules/pull/26/

mcollina commented:
I’m -1 for obvious reasons. I was traveling most of this week, and I _thought_ I filled in that doodle. The fact that people would have been kicked out was not written in the text of an issue, just in a single comment and maybe in meeting notes. Considering the amount of messages this group generates in a week, it was very easy to miss.

We are typically very sensitive about these things in this organization, so I encourage everyone to reconsider this approach.

joyeecheung commented:
By the way: you can create child teams under @nodejs/modules , and put observers in some where like @nodejs/modules-observers, non-observers in @nodejs/modules-members (with write access to this repo), so you can still ping everyone with the parent team, and sync the lists using ncu-team, and when there is a PR you can tell the difference by the greeness of their sign-offs.

joyeecheung commented:
Also...@nodejs/automation may help developing a tool that count votes by a specific team, now come to think of it.

benjamingr commented:
> I’m -1 for obvious reasons. I was traveling most of this week, and I thought I filled in that doodle. The fact that people would have been kicked out was not written in the text of an issue, just in a single comment and maybe in meeting notes. Considering the amount of messages this group generates in a week, it was very easy to miss.

There were at least 5 pings about this - and Myles reached out to everyone who hasn't filled out the doodle by email (I think). That said, I think the idea was mainly to not have people who didn't have time to participate in the team given the voting process and consensus model.

I'm personally fine with a rule like "observers who participate in two straight meetings and in issues are moved back to member status" and "members who don't participate in two meetings or the issue tracker are moved to observer status".

weswigham commented:
@benjamingr Maybe add your proposal to #8 so we remember to consider it in the next meeting?

mcollina commented:
Let me be very straightforward: Myles ping me only right before this PR was filed via twitter DM. I received no other direct ping.

And I just thought the doodle in question was https://t.co/TWYtjwpE5y. My mistake.

jdalton commented:
@mcollina The PR isn't merged (and even if so) if there was a misunderstanding I'm sure things could get ironed out. The "take my toys and go home" tone isn't really needed here. No one is against you. No one is out to get you. The doodle bit was confusing. It almost tripped up Sean too. No worries.

mcollina commented:
> The PR isn't merged (and even if so) if there was a misunderstanding I'm sure things could get ironed out. 

This PR has already been acted upon, as I'm no longer part of the group. It was not a question, but rather a representation of reality. Making it a question would have been enough.

> The "take my toys and go home" tone isn't really needed though. No one is against you. No one is out to get you.

Absolutely! I never thought that was the case. I was upset, and my tone was a bit over where it should have been.

> The doodle bit was confusing (it almost tripped up Sean too). 

When membership is considered, there should be nothing to "trip". It should be crystal clear if someone is in or out. People take time off, go on vacation, and might need to take some "away" time because of burnout. __We should not decide on a Wednesday that someone has untill Sarturday to fill something or they are out.__ This sets a bad precedent for this group, and the whole Node.js project. We can do better.
This is what I'm referring about in https://github.com/nodejs/modules/issues/3.

> I say no worries if you want on the team members list.

The group can decide that no, a one week period is not enough to remove a member. Someone might be off for a month _at least_. Then, ask them if they are still committed and would like to step down voluntarily, and after some time remove them if there is no answer. Everyone should have the chance to make that decision voluntarily first.
This is my proposal.

benjamingr commented:
@mcollina would you perhaps be interested in being a member of this team again and the decision on the removal discussed in the next meeting?

mcollina commented:
@benjamingr I care that people are allowed enough personal time away before being removed, and that should be asked directly several times. I would probably join back in some time if this passes. I still think this group work is critical for our community.

MylesBorins commented:
In retrospect, and based on comments in this thread and https://github.com/nodejs/modules/issues/8 I would like to suggest that we revisit the decision to move the eight members removed by this PR. This is already tagged for discussion in the upcoming meeting, but perhaps we can work out some of the ideas in #8 and put the findings in this PR before we discuss on the 28th.

@nodejs/modules thoughts?

@joyeecheung I've gone ahead and refactored the groups so that we have a modules team, with two subteams. Everyone in this PR who is currently moved to observers are in the observers subteam. With the permission of @mcollina I've added him to that team and back out the earlier commit from  @benjamingr 

Apologies again for how this turned out, especially on a long weekend.

abouthiroppy commented:
@MylesBorins 

I apologize for the late reply.
Should I join Node.js Foundation Modules Team Meeting? I can't join because it will be held at 4 AM at Tokyo.

joyeecheung commented on README.md:
```diff
@@ -17,41 +17,50 @@ Work includes:
 
 <!-- ncu-team-sync.team(nodejs/modules) -->
```
You can change this to `<!-- ncu-team-sync.team(nodejs/modules-collaborators) -->`, and use another `<!-- ncu-team-sync.team(nodejs/modules-observers) -->` block in the next section. `ncu-team` would sync both of them when running `ncu-team sync README.md`
